adserver/views.py

```diff
@@ -751,6 +753,152 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class AdvertiserGeoReportView(AdvertiserAccessMixin, GeoReportMixin, BaseReportView):
+
+    """A report for an advertiser broken down by geo."""
+
+    model = GeoImpression
+    template_name = "adserver/reports/advertiser-geo.html"
```

Comment:
Should probably do `s/_/-/` for the publisher report paths, since I used underscores.